src/Utils/CliMsgFormatter.php

```diff
@@ -118,7 +118,7 @@ public function progressCompact( int $i, int $total, ?int $current = null, ?int
 			$last = $total;
 		}
 
-		$progress = min( 100, round( ( $i / $total ) * 100 ) );
+		$progress = min( 100, round( ( $total ? $i / $total : 1 ) * 100 ) );
```

Comment:
Don't put a ternary in middle of the code. It makes it hard to understand, especially without parenthesis to clarify the processing order. It would be better to use if-statement or something like ` $progress = !$total ? 100 : min( 100, round( ( $i / $total ) * 100 ) );`

Reply:
Thanks, done.